src/spikeinterface/core/numpyextractors.py

```diff
@@ -94,37 +99,63 @@ def get_traces(self, start_frame, end_frame, channel_indices):
 
 
 class NumpySorting(BaseSorting):
+    """
+    In memory sorting object.
+    The internal representation is always done with a long "spike vector".
+
+
+    But we have convinient function to instantiate from other sorting object, from time+labels,
```

Comment:
convenience*

It would be even better if we added the specific methods that you mentioned. I can add them later after we merge as well.

Reply:
There are already. They are class method. I enhance the doc.